human_analysis/archive/3_gene_DE_analysis.ipynb

```diff
@@ -0,0 +1,1771 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sns.distplot(template_simulated_DE_stats[('ranking',\"\")].values, kde=False)"
```

Comment:
Is this just showing that each ranking is roughly equally common across the different simulated datasets?

Reply:
This is a debug statement to show that ranking should be equally distributed across genes in this case